docs/api-reference/next.config.js/turbopack-loaders.md

```diff
@@ -0,0 +1,41 @@
+---
+description: Configure Next.js with Turbopack to load Webpack loaders
+---
+
+# Turbopack Loaders (`next --turbo` only)
+
+Currently, Turbopack supports a subset of Webpack's loader API, allowing you to use some Webpack loaders to transform code in Turbopack.
+
+> **Warning**: This feature is experimental and will only work with `next --turbo`
+
+To configure loaders, add the names of the loaders you've installed and any options in `next.config.js`:
+
+```js
+module.exports = {
+  experimental: {
+    turbopackLoaders: {
+      '.mdx': [
+        {
+          loader: '@mdx-js/loader',
+          options: {
+            format: 'mdx',
+          },
+        },
+      ],
+      '.foo': 'my-foo-loader',
```

Comment:
Can you add docs explaining the key here? Is it a file extension? Does this support regex?